### DISCUSSION TITLE
I lost my account for creating an organisation

### DISCUSSION THREAD

**Discussion by mishankius** [community/community#64336] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I lost my account for creating an organisation, I created an organisation on my account, I made my alt account as the organiser, which I don't have access to. I can't get into the account where I created the organisation, what should I do?

**Comment by byrneh** [comment]
Raise a ticket to GitHub Support

**Reply by TheOriginalPiff** [reply]
What’s the account name?

**Comment by mutasim77** [comment]
Contact GitHub Support: https://support.github.com/
